pipelines/concourse.yml

```diff
@@ -1104,6 +1104,28 @@ jobs:
     - get: bbr-sdk-release
     - get: prod
     - get: ci
+  - put: bosh-topgun-worker-deployment
+    tags: [topgun]
+    params:
+      manifest: cbd/cluster/external-worker.yml
+      stemcells:
+      - gcp-xenial-stemcell/*.tgz
+      releases:
+      - concourse-release/*.tgz
+      - bpm-release/*.tgz
+      ops_files:
+      vars_files:
+      - cbd/versions.yml
```

Comment:
I think we're gonna want to deploy RCs, so can we maybe drop this and just add the relevant values below? In fact, the default for `concourse_version` at least might be `latest`, which would be what we want in this case.

Reply:
Added an ops-file that switches the versions over to `latest`, `dev-versions.yml`